index.html

```diff
@@ -166,13 +166,21 @@ <h2>
           <dd>
             <p>Prompts the user for permission to live-capture their display.</p>
             <p>
-              The user agent MUST let the end-user choose which display surface to share
-              out of all available choices every time, and MUST NOT use
-              <var>constraints</var> to limit that choice. Instead,
-              <var>constraints</var> MUST be applied to the media chosen by the user,
-              only after they have made their selection. This prevents an application
-              from influencing the selection of sources, see <a href="#constraints"></a>
-              for details.
+              The user agent MUST let the end-user choose which [=display surface=] to share
+              out of all available choices every time, and MUST NOT use <var>constraints</var>
+              to limit that choice.
+            </p>
+            <p>
+              The user agent MAY allow the application to use the {{displaySurface}} constraint to
+              signal a preference for a specific [=display surface=] type. The user agent MUST still
+              offer the user unlimited choice of any [=display surface=], but MAY order the sources
+              offered to the user according to this constraint.
+              It is advised that allowing applications to nudge users towards sharing a monitor
+              poses risks to user privacy. User agent may wish to ignore such hints.
```

Comment:
I would like the normative SHOULD we discussed. Something like:
```suggestion
              The user agent is strongly recommended to nudge users away from sharing a monitor,
              as this poses risks to user privacy.
```